### PR TITLE
Add many missing override keywords

### PR DESCRIPTION
**Pull request by peterbarker**
This helps corrects builds with -Werror=suggest-overrides

### CODE REVIEW

**Comment by peterbarker** [comment]
@googlebot I signed it!

File: googletest/include/gtest/gtest-matchers.h [hunk 1]
```diff
@@ -474,13 +474,13 @@ class PolymorphicMatcher {
    public:
     explicit MonomorphicImpl(const Impl& impl) : impl_(impl) {}
 
-    virtual void DescribeTo(::std::ostream* os) const { impl_.DescribeTo(os); }
+    virtual void DescribeTo(::std::ostream* os) const override { impl_.DescribeTo(os); }
```
Comment (asoffer) [inline googletest/include/gtest/gtest-matchers.h]: Here and everywhere else, remove `virtual`, as per the google style guide:
https://google.github.io/styleguide/cppguide.html#Inheritance

Thanks!

Comment (peterbarker) [reply googletest/include/gtest/gtest-matchers.h]: Done in #2475 - can't push freely to the source branch for this PR.

**Comment by peterbarker** [comment]
Replaced by #2475

**Pull request closed without merging**